lib/core/di/firebase_module.dart

```diff
@@ -14,4 +14,32 @@ Future _registerFirebaseModule() async {
   //
   // /// Remote Config
   _getIt.registerSingleton(FirebaseRemoteConfig.instance);
+
+  // not sure this should go here?
```

Comment:
Hum, i am not sure if FirebaseRemoteConfig.instance is creating a new instance? But either way, we should move this config outside of the module. 

I can handle this if you want.

Reply:
Yes please feel free to move this where it belongs.